internal/jobqueue/job.go

```diff
@@ -87,7 +87,7 @@ func (job *Job) Run() error {
 			}
 
 			/* TODO: communicate back the AMI */
-			_, err = a.Register(result.OutputID, options.Bucket, options.Key)
+			_, err = a.Register(options.ImageName, options.Bucket, options.Key)
```

Comment:
I would prefer `ImageName` to be part of `Target` struct to make things easier. API requires each upload to have its own name, so it isn't anything special to AWS.

Reply:
Good point!